<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>SignUp</class>
 <widget class="QDialog" name="SignUp">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>507</width>
    <height>377</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <property name="styleSheet">
   <string notr="true">#SignUp{
background-image : url(:/Images/Resources/Images/SignIn.jpg);
}</string>
  </property>
  <widget class="QWidget" name="layoutWidget">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>30</y>
     <width>193</width>
     <height>137</height>
    </rect>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout_3">
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout">
      <item>
       <layout class="QVBoxLayout" name="verticalLayout">
        <item>
         <widget class="QLabel" name="label">
          <property name="styleSheet">
           <string notr="true">QLabel{
	color : white
}
</string>
          </property>
          <property name="text">
           <string>User Name:</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLabel" name="label_2">
          <property name="styleSheet">
           <string notr="true">QLabel{
	color : white
}
</string>
          </property>
          <property name="text">
           <string>Password:</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLabel" name="label_3">
          <property name="styleSheet">
           <string notr="true">QLabel{
	color : white
}
</string>
          </property>
          <property name="text">
           <string>Password:</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLabel" name="label_4">
          <property name="styleSheet">
           <string notr="true">QLabel{
	color : white
}
</string>
          </property>
          <property name="text">
           <string>Email:</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QVBoxLayout" name="verticalLayout_2">
        <item>
         <widget class="QLineEdit" name="lineEdit_S_UserName">
          <property name="styleSheet">
           <string notr="true">QLineEdit{color:black;}</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLineEdit" name="lineEdit_S_Password">
          <property name="styleSheet">
           <string notr="true">QLineEdit{color:black;}</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLineEdit" name="lineEdit_S_Password_Re">
          <property name="styleSheet">
           <string notr="true">QLineEdit{color:black;}</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLineEdit" name="lineEdit_S_Email">
          <property name="styleSheet">
           <string notr="true">QLineEdit{color:black;}</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
     </layout>
    </item>
    <item>
     <widget class="QPushButton" name="pushButton_S_SignUp">
      <property name="styleSheet">
       <string notr="true">QPushButton {
    font-size: 12pt;
    font-weight: bold;
    color: black;
}</string>
      </property>
      <property name="text">
       <string>Sign Up</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
